code/datums/gamemodes/distress.dm

```diff
@@ -539,3 +539,12 @@
 /datum/game_mode/distress/spawn_larva(mob/xeno_candidate, mob/living/carbon/xenomorph/mother)
 	var/datum/hive_status/normal/HS = GLOB.hive_datums[XENO_HIVE_NORMAL]
 	return HS.spawn_larva(xeno_candidate, mother)
+
+/datum/game_mode/distress/proc/on_xeno_evolve(datum/source, mob/living/carbon/xenomorph/new_xeno)
+	if(SSticker.round_start_time + SHUTTLE_HIJACK_LOCK > world.time)
+		if(XENO_TIER_ONE)
```

Comment:
There's a `switch()` lacking here.

Reply:
Please don't resolve issues on your own. This one is not properly resolved.
Look for an example where you took this code from, of the correct `switch()` value to use here.